internal/cmd/gocdk/internal/static/static.go

```diff
@@ -132,8 +133,27 @@ func AddProvider(provider string) *Action {
 	}
 }
 
+// ReadLocal reads a local Terraform variable from main.tf.
+// The local variable was likely added using an Action returned from AddLocal.
+// If the variable is not found, it returns the empty string and no error.
+func ReadLocal(biomeDir, key string) (string, error) {
```

Comment:
Organization: This function doesn't feel like it conceptually belongs under the heading "static files" to me. Perhaps we should change the package name to reflect, but probably doesn't belong in this PR.

Reply:
Agreed; added a TODO.